x/time/time.go

```diff
@@ -0,0 +1,17 @@
+package time
+
+import "time"
+
+// UnixNano is used to indicate that an int64 stores a unix timestamp at
+// nanosecond resolution
+type UnixNano int64
+
+// Time returns a time.Time from a UnixNano
+func (u UnixNano) Time() time.Time {
```

Comment:
nit: `ToTime` might be a better name?

Reply:
Yeah thats actually what I called it originally, I can change it